From: Sven Eckelmann <sven@narfation.org>
Date: Tue, 5 Dec 2017 19:18:48 +0100
Subject: batctl: upgrade package to latest release 2017.4

* synchronization of batman-adv netlink header
* coding style cleanups and refactoring
* documentation cleanup
* bugs squashed:

  - improve error handling for libnl related errors
  - add checks for various allocation errors

Signed-off-by: Sven Eckelmann <sven@narfation.org>

diff --git a/batctl/Makefile b/batctl/Makefile
index 2a1912550ab2251a1999cc3c5b2ec380d6d15f2d..b3647142b79a0a748eab04d52bf167c87ca19010 100644
--- a/batctl/Makefile
+++ b/batctl/Makefile
@@ -9,10 +9,10 @@ include $(TOPDIR)/rules.mk
 
 PKG_NAME:=batctl
 
-PKG_VERSION:=2017.3
+PKG_VERSION:=2017.4
 PKG_RELEASE:=0
-PKG_MD5SUM:=a4219f90b077a874d207cb0973d4d3cf
-PKG_HASH:=29c7f075023e02ed9a9299008ec73cfdb4e1b04991d27e4f396131b4059b88a8
+PKG_MD5SUM:=142d5b0b4a9efebfe781ed28beb7d7ce
+PKG_HASH:=047d6aac4da51dbfb5721df0bbc9b1699c8431e4707157f939c6d6c4f214e464
 
 PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
 PKG_SOURCE_URL:=https://downloads.open-mesh.org/batman/releases/batman-adv-$(PKG_VERSION)
